### ISSUE TITLE
Textarea raises a warning when used with validation

### ISSUE THREAD

**Issue by nstoychev** [telerik/kendo-react#791]
Warning: Received `true` for a non-boolean attribute `visited`.

example:  https://stackblitz.com/edit/react-1rn1rv?file=app/main.jsx

**Comment by vladimirivanoviliev** [comment]
Actually this warning is due to incorrect demo. The `visited` prop should be destructed from `FieldRenderProps`:

```
  const { validationMessage, visited, touched, label, optional, id, valid, disabled, hint, ...others } = fieldRenderProps;
```

**Comment by simonssspirit** [comment]
Reported again in 1501040

**Comment by Xizario** [comment]
Fixed in 4.3.0
